src/ucs/config/parser.h

```diff
@@ -420,16 +424,22 @@ ucs_status_t ucs_config_parser_set_value(void *opts, ucs_config_field_t *fields,
                                          const char *name, const char *value);
 
 /**
- * Check all UCX_ environment variables have been used so far by the
- * configuration parser, issue a warning if not. Called just before program exit.
+ * Check all environment variables starting from prefix that have been used so far
+ * by the configuration parser, issue a warning if not. Called just before
+ * program exit.
+ *
+ * @param prefix     Environment variable prefix.
  */
-void ucs_config_parser_warn_unused_env_vars();
+void ucs_config_parser_warn_unused_env_vars(const char *prefix);
```

Comment:
the argument name for some functions is "prefix" and others is "env_prefix", are they the same? name should be consistent..